src/main/ui/BrewYah.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {BrowserRouter, Switch, Route, Link} from 'react-router-dom';
+import Braumeister from "braumeister";
+import Taproom from "taproom";
+
+class BrewYah extends React.Component {
+    render(props) {
+        return (<BrowserRouter>
```

Comment:
Top level `BrowserRouter` allows us to remove the Routers from the child UI Components.